CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 ## [unreleased]
 ### Added
 - Button on SMN CN page to search variants within SMN1 and SMN2 genes
-- Options for selectively updating OMICS variants (fraser, outrider) on a case
 ### Changed
 - Documentation for OMICS variants and updating a case
+### Fixed
+- Options for selectively updating OMICS variants (fraser, outrider) on a case
```

Comment:
Hm, this is still an added methinks: new options to the CLI.

Reply:
moved to added!